terraform/container-definitions/worker_container_definition.json

```
@@ -100,6 +100,10 @@
         "name": "AUDIT_FEEDBACK_WORKSHEET_GID",
         "value": "${audit_feedback_worksheet_gid}"
       },
+      {
+        "name": "AUDIT_EXPRESS_INTEREST_WORKSHEET_GID",
+        "value": "audit_express_interest_worksheet_gid"
```

```suggestion
        "value": "${audit_express_interest_worksheet_gid}"
```

So that the value of `audit_express_interest_worksheet_gid` gets rendered instead of the string